apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```
@@ -475,6 +475,11 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 		}
 	}
 
+	// Tweak template title (post-title) component.
+	.template-title {
```

I do wonder whether this selector is too "global". The eslint rule tries to protect us from this by requiring highly specific classnames that confirm to the component namespace. I can see we've disabled this rule for the entire render so we're missing this guidance where it could be helpful.

I'd say we should make this select much more targetted so there is no danger of anyone else adding a similar CSS class that could overload / corrupt these styles. 

```suggestion
	.block-iframe-preview__template-title {
```